### ISSUE TITLE
Erro no valor

### ISSUE THREAD

**Issue by flaviolenz** [bacen/pix-api#248]
Esse erro ta sinistro.
Qr estáticos nos valores de R$198 e R$100.
PSP recebedor relatou que recebeu R$1 e R$1,98
PSP pagador emitiu ate comprovante de R$100 e R$198

Quem tava errado?

To achando que o mais seguro eh colocar o .00 nos QR, pra garantir.

**Comment by flaviolenz** [comment]
![Screenshot_20201209-220609_WhatsApp](https://user-images.githubusercontent.com/10598781/101707830-c91d0a00-3a6a-11eb-89c6-0b7f87655884.jpg)

**Comment by rubenskuhl** [comment]
Sim, apesar de segundo o padrão ser indiferente, é sim mais seguro usar "1.00" do que "1.0" ou "1" em termos de correta aceitação pelos pagadores. E também é mais seguro especificar o valor no QR-Code dinâmico mesmo ele sendo em teoria desnecessário.

**Comment by flaviolenz** [comment]
![Screenshot_20201209-220811_Gerencianet](https://user-images.githubusercontent.com/10598781/101707947-0d100f00-3a6b-11eb-8ddb-097a6787441b.jpg)

**Comment by flaviolenz** [comment]
Qrcode foi esse:

00020126580014br.gov.bcb.pix01365a5ea8c2-cbb5-4c25-a7be-2a778871fc5f52040000530398654031985802BR5911Arrumadinho6008Brasilia62290525e433v1530u155f31mSushiPro63048381

**Comment by renatofrota** [comment]
O erro pode estar de qualquer um dos dois lados. Mas como no comprovante do Santander (PSP Pagador) os R$ 198 apareceram sem os centavos, eu suponho que ele enviou "198" para o PSP recebedor (que considerou - preventivamente ou por regulamentação* - como sendo centavos).

* acho que li em algum lugar que a comunicação entre os PSPs/BACEN se dá com os valores em centavos, diferentemente dos QR Codes e da API Pix.

**Comment by flaviolenz** [comment]
o erro foi do PSP Pagador.
ele deu o recibo com 198 mas debitou so 1,98 na conta (e transferiu 1,98 pro PSP Recebedor)

**Comment by dev-gto** [comment]
Sim, deveria considerar R$198,00, e não o valor em centavos.
Do EMV:
```
4.7.4 Transaction Amount (ID "54")

4.7.4.1 If present, the Transaction Amount shall be different from zero, shall only include
(numeric) digits "0" to "9" and may contain a single "." character as the decimal
mark. When the amount includes decimals, the "." character shall be used to
separate the decimals from the integer value and the "." character may be present
even if there are no decimals.
The number of digits after the decimal mark should align with the currency
exponent associated to the currency code defined in [ISO 4217].
The above describes the only acceptable format for the Transaction Amount. It
cannot contain any other characters (for instance, no space character can be used
to separate thousands).
The following are examples of valid Transaction Amounts: "98.73", "98" and "98.".
The following are NOT valid Transaction Amounts: "98,73" and "3 705".

4.7.4.2 The Transaction Amount shall not be included if the mobile application should prompt
the consumer to enter the amount to be paid to the Merchant.
```
